<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Michael Reitman</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Qualtrics Downloader</string>
  </property>
  <widget class="QWidget" name="QualtricsImporter">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>150</number>
      </property>
      <property name="rightMargin">
       <number>150</number>
      </property>
      <item row="2" column="0">
       <widget class="QPushButton" name="InputConfig_button">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;select your .txt configuration file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Choose config file</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="ChooseOutput_button">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;choose where to download the survey responses&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Choose output directory</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="DownloadQualtrics_button">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;download the Qualtrics survey responses&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Download Qualtrics responses</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="CreateConfig_button">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;create a blank .txt file of Qualtrics IDs&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Create config file</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="ChooseCsvFile_button">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Choose CSV file</string>
        </property>
        <property name="text">
         <string>Choose CSV file</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="CsvFileTxt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string>Path to your CSV file</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QPushButton" name="LoadCsvFile_button">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Load CSV file</string>
        </property>
        <property name="text">
         <string>Load CSV file</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QComboBox" name="ColumnNameTxt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="Plot_button">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Plot responses from downloaded file</string>
        </property>
        <property name="text">
         <string>Plot</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ConfigFileTxt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>IBeamCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="placeholderText">
         <string>Path to your Config (.txt) file</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QProgressBar" name="progressBar">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="OutputDirectoryTxt">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Path to your output directory</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionFyQIDs"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFyQIDs">
   <property name="text">
    <string>Find your Qualtrics IDs</string>
   </property>
  </action>
  <action name="actionOpen_Qualtrics_Plotter">
   <property name="text">
    <string>Open Qualtrics Plotter</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>CreateConfig_button</tabstop>
  <tabstop>InputConfig_button</tabstop>
  <tabstop>ConfigFileTxt</tabstop>
  <tabstop>ChooseOutput_button</tabstop>
  <tabstop>OutputDirectoryTxt</tabstop>
  <tabstop>DownloadQualtrics_button</tabstop>
  <tabstop>ChooseCsvFile_button</tabstop>
  <tabstop>CsvFileTxt</tabstop>
  <tabstop>LoadCsvFile_button</tabstop>
  <tabstop>ColumnNameTxt</tabstop>
  <tabstop>Plot_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>InputConfig_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChooseConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChooseOutput_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChooseOutput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CreateConfig_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>CreateConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DownloadQualtrics_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>DownloadQualtrics()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DownloadQualtrics_button</sender>
   <signal>clicked()</signal>
   <receiver>progressBar</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>1018</x>
     <y>640</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChooseCsvFile_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>ChooseCsvFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LoadCsvFile_button</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>LoadCsvFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Plot_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Plot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CreateConfig()</slot>
  <slot>ChooseConfig()</slot>
  <slot>ChooseOutput()</slot>
  <slot>DownloadQualtrics()</slot>
  <slot>ChooseCsvFile()</slot>
  <slot>LoadCsvFile()</slot>
  <slot>Plot()</slot>
  <slot>UpdateText()</slot>
 </slots>
</ui>
